simde/x86/avx512bw.h

```diff
@@ -1930,6 +1930,46 @@ simde_mm512_sad_epu8 (simde__m512i a, simde__m512i b) {
   #define _mm512_sad_epu8(a, b) simde_mm512_sad_epu8(a, b)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m512i
+simde_mm512_sll_epi16 (simde__m512i a, simde__m128i count) {
+  #if defined(SIMDE_X86_AVX512BW_NATIVE)
+    return _mm512_sll_epi16(a, count);
+  #else
+    simde__m512i_private
+      r_,
+      a_ = simde__m512i_to_private(a);
+
+    #if SIMDE_NATURAL_VECTOR_SIZE_LE(256)
+        for (size_t i = 0 ; i < (sizeof(r_.m256i) / sizeof(r_.m256i[0])) ; i++) {
```

Comment:
There is some extra indenting here.  It's not a big deal, and I'll fix it when I merge, but you might want to think about installing an [EditorConfig](https://editorconfig.org/) plugin so your editor picks up some configuration settings.  IIRC you use CLion, for which there is [a plugin](https://github.com/JetBrains/intellij-community/tree/master/plugins/editorconfig).

Reply:
I didn't know about the plugin for CLion, I have installed it now.
Thank You